CONTRIBUTING.md

```
@@ -0,0 +1,257 @@
+
+```bash
+git add .
+git commit -m "feat: add user authentication"
+# or
+git commit -m "fix: resolve navigation bug"
+```
+
+**Commit Message Format:**
+- `feat:` for new features
+- `fix:` for bug fixes
+- `docs:` for documentation changes
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)